addons/binding/org.openhab.binding.homematic/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,41 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Homematic Binding
+Bundle-SymbolicName: org.openhab.binding.homematic;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/nanohttpd-2.3.0.jar
```

Comment:
One very last wish: Since you are including a 3rd party library, could you add an about.html to the bundle (similar to what I requested in https://github.com/openhab/openhab2-addons/pull/700#issuecomment-213535801)? Thanks!